datafusion/core/tests/parquet/row_group_pruning.rs

```diff
@@ -336,16 +369,38 @@ async fn prune_int32_eq_in_list() {
 #[tokio::test]
 async fn prune_int32_eq_in_list_2() {
     // result of sql "SELECT * FROM t where in (1000)", prune all
-    test_prune(
+    // test whether statistics works
+    test_prune_verbose(
         Scenario::Int32,
         "SELECT * FROM t where i in (1000)",
         Some(0),
+        Some(0),
         Some(4),
         0,
     )
     .await;
 }
 
+#[tokio::test]
+async fn prune_int32_eq_large_in_list() {
+    // result of sql "SELECT * FROM t where i in (2050...2582)", prune all
+    // test whether sbbf works
+    test_prune_verbose(
+        Scenario::Int32Range,
+        format!(
+            "SELECT * FROM t where i in ({})",
+            (200050..200082).join(",")
+        )
+        .as_str(),
+        Some(0),
+        Some(1),
+        // we don't support pruning by statistics for in_list with more than 20 elements currently
```

Comment:
Also I think it the pruning by bloom filters happens first, so it may not even try to prune by statistics

👍